### PR TITLE
Added transaction finder to make more easy the usage of claimer

### PR DESCRIPTION
**Pull request by notf0und**
Used code base from claimer, removed unnecessary code and developed the method to check get the transaction id's from a list of addresses in a file. The script check every single address and get the balance claimable for each fork.

### CODE REVIEW

**Comment by avialias** [comment]
are this changes merged to ymgve master branch...
it would help with that txt file which holds list of public addresses...

**Comment by avialias** [comment]
Please resolve confflicts and merge to ymgve master fork

**Comment by avialias** [comment]
Or can you just ELI5 me how to use your code to automate claim from 30 priv keys?

I have problem with getting right and last Trx before fork block.
For each coin is a little different.

Please help me with an example with 2-3 test priv key.....step by step please....please

**Comment by notf0und** [comment]
Sorry, I was out and recently seen this. The usage is putting all addresses on a file separated by a line break and then:
claymer.py <cointype> <file> <destination address>

**Comment by StJoshua** [comment]
Resolve. So we can all bank on it

**Comment by daedalus** [comment]
Bump!

**Comment by javiertuerca** [comment]
Como estas:
Estoy utilizando windows 7 y me arroja el siguiente error:

Traceback (most recent call last):
  File "C:\bitcoin_fork_claimer\claimer.py", line 1096, in <module>
    addresses = [address.rstrip('\n') for address in open(args.addrfile)]
IOError: [Errno 2] No such file or directory: 'C:\\bitcoin_fork_claimer\\lista.txt'

La carpeta bitcoin_fork_claimer esta ubicada en C: junto con la lista,txt en su interior.
como puedo resolverlo ?
muchas gracias
